extension/src/unreal-debugclient.ts

```diff
@@ -374,7 +374,7 @@ export interface UnrealDataBreakpoint
 
 export function setDataBreakpoints(breakpoints: UnrealDataBreakpoint[])
 {
-    let count = breakpoints.length <= 4 ? breakpoints.length : 4;
```

Comment:
The reason for this change is that inactive Data Breakpoints on stack variables would not be deactivated when the stack frame goes out of scope, this way we catch even inactive Data Breakpoints. However we will unfortunately not catch disabled Data Breakpoint (i.e the user has unchecked the checkbox in the UI) since those arent provided to the Debug Adapter at all from what I could see. If the UI has two Data Breakpoint and one is disabled, the array of breakpoints the Debug Adapter receives will contain 1 element.